SPECS/kata-containers-cc/kata-containers-cc.spec

```diff
@@ -289,6 +289,12 @@ install -D -m 0755 %{_builddir}/%{name}-%{version}/tools/osbuilder/image-builder
 %exclude %{osbuilder}/tools/osbuilder/rootfs-builder/ubuntu
 
 %changelog
+* Wed May 29 2024 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 3.2.0.azl2-1
+- Auto-upgrade to 3.2.0.azl2
+
+* Mon May 20 2024 Saul Paredes <saulparedes@microsoft.com> - 3.2.0.azl1-2
```

Comment:
qq - usually one changelog entry per new PR. Is this intended? I do not see 3.2.0.azl1-2 to be on CBL-Mariner/main...

Reply:
Good catch!